addons/event/data/mail_template_data.xml

```diff
@@ -98,16 +98,25 @@
                                 <img src="/web_editor/font_to_img/61555/rgb(81,81,102)/34" style="padding:4px;max-width:inherit;" height="34" alt=""/>
                             </td>
                             <td style="padding: 0px 10px 0px 10px;width:50%;line-height:20px;vertical-align:top;">
-                                <div>
-                                    <strong>From</strong>
-                                    <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "date_only": True, "tz_name": object.event_id.date_tz, "format": "long"}'>May 4, 2021</t>
-                                     - <t t-out="object.event_id.date_begin" t-options='{"widget": "datetime", "time_only": True, "tz_name": object.event_id.date_tz, "hide_seconds": True, "format": "short"}'>7:00 AM</t>
```

Comment:
Oy @Clement-Cardot Ah yes, there are those too...
To keep unit commits, and to clarify what we do in this PR, we should probably do it in this order:
- First, a commit doing only all the updates on existing templates with hide_seconds option, and that removes the computed fields.
- Then, second, all the real changes (including here the addition of "If one day we do this, else we do that" since it was not there before)

We did a lot of aller-retours in this validation so if you're busy with something else no worries, I can do it later myself :)

Cheers

Reply:
I think that I managed to separate the changes in the two commits ^^

Reply:
:partying_face: Thanks! Not the funniest but looking great :)